### PR TITLE
include new domains

### PR DESCRIPTION
**Pull request by songgao**
will vendor into KBFS once this is merged so we don't have multiple code snippet handling the same logic.

### CODE REVIEW

File: go/libkb/ca.go [hunk 1]
```diff
@@ -3,10 +3,35 @@
...
+		}
+		return []byte(apiCA), true
+
+	// Staging CA.
+	case strings.HasSuffix(host, "dev.keybase.io"),
+		strings.HasSuffix(host, "dev.keybaseapi.com"):
+		return []byte(KBFSDevCA), true
+
+	// Prod CA.
+	case strings.HasSuffix(host, "kbfs.keybase.io"),
```
Comment (mmaxim) [inline go/libkb/ca.go]: Needs `core.keybase.io` right?

Comment (songgao) [reply go/libkb/ca.go]: good catch; will update!

Comment (songgao) [reply go/libkb/ca.go]: actually it seems core.keybase.io uses the CA for api.keybase.io. Is that correct?

Comment (maxtaco) [reply go/libkb/ca.go]: not for gregor. this was our major confusion

Comment (mmaxim) [reply go/libkb/ca.go]: Could be a problem with this change then, since Gregor could be on that address and will be using KBFS CA.

Comment (songgao) [reply go/libkb/ca.go]: I'll just bundle both CA in then.

**Comment by songgao** [comment]
Does latest commit look good to you? @mmaxim @maxtaco

**Comment by mmaxim** [comment]
It's hard for me to tell if it will just work, but I guess it looks ok.

**Comment by songgao** [comment]
I tested locally and both client and KBFS still work, so I guess it's safe to merge in the sense that it wouldn't break stuff ...

**Comment by songgao** [comment]
Thanks for the thumb! Will merge this along with KBFS one once Jenkins passed (so that CircleCI will be happy on master).

**Pull request merged**